forem/forem
# [Moderation] Allow admins to delete comments

## What type of PR is this? (check all applicable)



## Description
This PR allows `admins` and `tech_admins` to delete comments on the comment-moderation page.
~**I'm having the following issues:**~
- ~The flash notice/alert does not appear after the controller actions~
- ~The confirm dialog does NOT appear after clicking the "Delete Button"~


## QA Instructions, Screenshots, Recordings

1. Make sure your user has `admin`, `super_admin` or `tech_admin` role. 
2. Fire up `localhost`, locate a comment, select "Moderate" from the three-dot menu.
3. On the comment moderation page, scroll down and click "Delete Comment".
4. You should see the comment moderation page, and the comment snippet replaced with `[deleted]`.

<img width="1440" alt="Screen Shot 2020-08-03 at 3 52 41 PM" src="https://user-images.githubusercontent.com/32520970/89318714-26342980-d64d-11ea-8ed7-d0dd4afd11d1.png">

<img width="1440" alt="Screen Shot 2020-08-03 at 3 52 08 PM" src="https://user-images.githubusercontent.com/32520970/89318729-2b917400-d64d-11ea-9401-767c9b460afa.png">


## Added tests?



## Added to documentation?



![Woman Looking Worried](https://media.giphy.com/media/3o6Ztcw7GVQyotIKhW/giphy.gif)
